src/main/java/org/openjdk/jextract/impl/ClassSourceBuilder.java

```diff
@@ -137,10 +135,12 @@ void appendIndentedLines(String s) {
         sb.appendIndentedLines(s);
     }
 
-    final void emitPrivateDefaultConstructor() {
-        appendLines(STR."""
-            // Suppresses default constructor, ensuring non-instantiability.
-            private \{className}() {}
+    final void emitDefaultConstructor() {
+        appendIndentedLines(STR."""
```

Comment:
Since I was here, I decided to fix indentation, as I noticed that this is always off - hopefully that's not disruptive with other ongoing work